kruize-clowdapp.yaml

```diff
@@ -134,6 +136,8 @@ objects:
               value: "UTC"
             - name: deletepartitionsthreshold
               value: "16"
+            - name: KRUIZE_PARTITION_INTERVAL
+              value: "0 0 * * *" # Run the cronjob every day at midnight. Earlier it was set for 16th day
```

Comment:
This should not be here. It should be under `parameters` section - https://github.com/RedHatInsights/ros-ocp-backend/blob/main/kruize-clowdapp.yaml#L204

Reply:
Good Catch! I overlooked at it. Thanks @patilsuraj767